src/cmd/prune.ts

```diff
@@ -0,0 +1,9 @@
+import prune from '../api/prune'
+import {PublicInstallationOptions} from '../api/install'
+
+export default (input: string[], opts: PublicInstallationOptions) => {
+  if (input && input.length) {
+    return prune(input, opts)
+  }
+  return prune(opts)
```

Comment:
> If we decide to change the order of args, I guess we have to move opts to the beginning in all the API functions for consistency

maybe, though i dont wanna be a hippo here aiming to change everything all over the place, so lets not do this until everybody agreed that this change leads to better dx